fastlane/Fastfile

```diff
@@ -273,15 +326,22 @@ platform :ios do
   end
 
   private_lane :edit_changelog do |options|
+    generated_contents = options[:generated_contents].to_s
+
     # Open CHANGELOG.latest.md in editor
     changelog_filename = "CHANGELOG.latest.md"
     changelog_path = File.absolute_path("../#{changelog_filename}")
     editor = ENV['FASTLANE_EDITOR'] || ENV['EDITOR'] || 'vim'
 
+    if generated_contents.size > 0
+      UI.message("Using auto generated contents:\n#{generated_contents}")
+      File.write(changelog_path, generated_contents)
+    end
```

Comment:
if size == 0, should we error out?

Reply:
Ummm, that is probably a good idea! I'll add that 💪